src/index.js

```diff
@@ -14,7 +15,8 @@ const errorWasThrown = {}
 const DEFAULT_OPTIONS = {
   moduleName: 'styled-components',
   importName: 'default',
-  strict: false
+  strict: false,
+  ignoreFiles: []
 }
 
 module.exports = options => ({
```

Comment:
Tests fail due to `options` is not normalized. A simple adjustment can be,

```
const processerFactory = options => ({
 ... ...
});
module.exports = (options) => processerFactory(Object.assign({}, DEFAULT_OPTIONS, options));
```

Reply:
I'm not sure what you mean by the `processerFactory` function.
But I mapped the options now and the tests are green

Reply:
`processerFactory` means the whole original codes, and please forgive my typo. It should be `processorFactory` or `realProcessor`. If writing in this way, you can handle options in the unique place.

```
const realProcessor = options => ({
  code() {},
  result() {}
});
module.exports = (options) => realProcessor(Object.assign({}, DEFAULT_OPTIONS, options));
```

Reply:
@schoenwaldnils Looks like we are almost there. Would you mind processing options like above and squashing commits to less than or equal to 3? Thanks for your work in advance.

Reply:
Now I understood what you meant with the processing ^^
Done